examples/energy-management-app/energy-management-common/src/DeviceEnergyManagementDelegateImpl.cpp

```diff
@@ -115,6 +115,8 @@ Status DeviceEnergyManagementDelegate::PowerAdjustRequest(const int64_t powerMw,
 
     SetESAState(ESAStateEnum::kPowerAdjustActive);
 
+    // mPowerAdjustCapabilityStruct is guaranteed to have a value as validated in Instance::HandlePowerAdjustRequest.
+    // If it did not have a value, this method would not have been called.
```

Comment:
OK, so on the delegate API, this function needs to be documented accordingly, no?  Callers must call `GetPowerAdjustmentCapability` and ensure the return value is not null before calling `PowerAdjustRequest`?

Reply:
The API documentation has been updated.